changelogs/CHANGELOG-1.4.md

```diff
@@ -1,5 +1,52 @@
 # Changelog since v1.3.0
 
+## v1.4.2
+
+Date: 2022-12-5
+
+### Changes
+
+- Remove volume health annotation from PVC before the backup. (#495, @xing-yang)
+- Removes health annotation from PVC before handing it over to Velero during the restore. (#498, @xing-yang)
```

Comment:
In the future, please add the "--markdown-links" flag when running release-notes so that the links will be auto-generated.

Reply:
Sure, I updated our internal release procedure docs.